controllers/v1/userProjects.js

```diff
@@ -1125,5 +1126,106 @@ module.exports = class UserProjects extends Abstract {
                 }
         })
     }
+    /**
+    * @api {post} /improvement-project/api/v1/userProjects/listPendingProjects
+    * Lists projects based on the status 
+    * @apiVersion 1.0.0
+    * @apiGroup User Projects
+    * @apiSampleRequest /improvement-project/api/v1/userProjects/listPendingProjects
+    * @apiParamExample {json} Response:
+    /**
+
+    /**
+     * Lists projects based on the status
+     * @method
+     * @name listPendingProjects
+     * @returns {JSON} Project details
+    */
+
+    async listPendingProjects(req) {
+        return new Promise(async (resolve, reject) => {
+            
```

Comment:
merge this into one controller as userOverview don't list pending or started or inprogress @borkarsaish65

Reply:
done